### ISSUE TITLE
conftest duplication between case_studies

### ISSUE THREAD

**Issue by ismael-mendoza** [prob-ml/bliss#488]
I noticed that the conftests in both of the `sdss_galaxies_*` case_studies are essentially the same to each other and also to the one in `tests`. Could we just use the one in `tests` instead of having repeated code? ( I had to disable linting in #487 due to the code duplication ) 

If the problem is that they are in different parentn directories one solution is to have a `tests/case_studies/sdss_galaxies_*` folders and that way all tests can use the same conftest in `tests`.

What do you think @dereklhansen ?

**Comment by dereklhansen** [comment]
Yes, I agree moving everything under `tests/` is best. pytest doesn't seem to handle well having tests scattered across multiple folders. And the VS Code pytest module doesn't currently work for tests outside `tests/`, and I like using it's debug feature
